scripts/order.py

```diff
@@ -2,7 +2,7 @@
 '''
 Audits logfiles to determine the parent of a derivative package.
 This script can aid in automating large accessioning procedures that involve
-the accessioning of derivatives along with masters, eg a Camera Card and
+the aipping of derivatives along with masters, eg a Camera Card and
```

Comment:
@raecasey do we need to rephrase there as well? Maybe to "the AIPing" or "the AIP creating"?

Reply:
yes, - 'the AIPing'

Reply:
Fixed.